Gemfile

```diff
@@ -39,6 +39,9 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails', '~> 6.1'
```

Comment:
What is the functionality of each of these gems?

Reply:
It allows you to define and use factories to manage test data, making them less error-prone, more explicit in testing, and easier to work within fixtures.